server/scripts/populate-test-data/populateFakeTestResults.js

```diff
@@ -115,34 +147,27 @@ const populateFakeTestResults = async (testPlanRunId, fakeTestResultTypes) => {
 
         index += 1;
     }
+
+    if (testPlanReport.runnableTests.length !== fakeTestResultTypes.length) {
```

Comment:
Can a comment be included on why this is being added as well? Outside the context of this PR, it may take a bit longer to follow on why, for any future maintenance.

Reply:
Comment added.